lib/ui/painting/image_decoder_impeller.cc

```diff
@@ -452,13 +452,8 @@ void ImageDecoderImpeller::Decode(fml::RefPtr<ImageDescriptor> descriptor,
           // texture is implemented on other platforms.
           sk_sp<DlImage> image;
           std::string decode_error;
-#ifdef FML_OS_IOS
-          std::tie(image, decode_error) = UploadTextureToPrivate(
-              context, bitmap_result.device_buffer, bitmap_result.image_info);
-#else
```

Comment:
Instead of this, you can use the GPU sync switch to conditionally use this path on iOS.

Reply:
You need to disable mipmaps as well

Reply:
I mentioned mipmaps in the description.

> The blit pass that generates the mipmaps still fails if the app is backgrounded, but not writing all the mip levels is safe for Metal and Vulkan -- GLES is the problem child.

Reply:
On some versions of iOS we've seen crashes when accessing command encoders in the background